src/styles/styles/card.scss

```diff
@@ -0,0 +1,146 @@
+  .catalog__content {
+    box-sizing: border-box;
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: repeat(1, $card-width);
+
+    justify-content: center;
+    gap: 46px 48px;
+    padding: 50px 40px ;
+    margin: 0 auto;
+
+    @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+    }
+
+    @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+    }
+
+    @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+    }
+  }
+
+  .card {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: $card-width;
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+    border: 1px solid $border-color;
+    border-radius: 5px;
+    transition: transform $transition-time;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+
+    &__img {
+      margin: auto;
+      width: 160px;
+      height: 134px;
+      margin-bottom: 40px;
+    }
+
+    &__title {
+      margin: 0;
+      margin-bottom: 4px;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $main-color;
+      transition: color $transition-time;
+    }
+
+    &:hover .card__title {
+      color: $card-title-hover;
+    }
+
+    &__code {
+      margin: 0;
+      margin-bottom: 16px;
+      font-size: 10px;
+      line-height: 14px;
+      color: $secondary-color;
+    }
+
+    &__reviews {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 24px;
+    }
+
+    &__review_point {
+      margin: 0;
+      font-size: 10px;
+      color: $main-color;
+    }
+
+    &__price {
+      display: flex;
+      justify-content: space-between;
+      margin: 0;
+      margin-bottom: 16px;
+    }
+
+    &__price-text {
+      margin: 0;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &__price-number {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-color;
+    }
+
+    &__buy-link {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      box-sizing: border-box;
+      border: 1px solid $button-color;
+      border-radius: 5px;
+      font-size: 14px;
+      font-weight: 700;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)